RecoEgamma/EgammaTools/src/GBRForestTools.cc

```diff
@@ -135,3 +156,13 @@ std::unique_ptr<const GBRForest> GBRForestTools::createGBRForest(const edm::File
 
   return gbrForest;
 }
+
+std::unique_ptr<const GBRForest> GBRForestTools::createGBRForest(const std::string &weightFile){
+  std::vector<std::string> varNames;
+  return GBRForestTools::createGBRForest(weightFile);
```

Comment:
this looks like an infinite recursion.
```return GBRForestTools::createGBRForest(weightFile, varNames); ``` instead ?

Reply:
Right, sorry...